app/models/station.rb

```diff
@@ -29,6 +30,13 @@ class Station < ActiveRecord::Base
   has_many :recent_observations, -> { order('created_at ASC').limit(10) }, class_name: 'Observation'
   has_one :current_observation, -> { order('created_at ASC').limit(1) }, class_name: 'Observation'
 
+  # Has scoped roles
+  # @see https://github.com/RolifyCommunity/rolify
+  resourcify
+
+  # Scopes
+  scope :visible, -> { unresponsive.merge(active) }
```

Comment:
Enums create scope methods - `unresponsive`  is a shortcut to `Station.where(status: :unresponsive)`.  `ActiveRecord::Relations` have a `.merge` method that merges one scope with another. However in this case it did not quite work as intended. What I wanted was 'WHERE stations.status IN (2,3) ' but this gives `'WHERE stations.status = 3`. It is fixed in the Pundit PR.